packages/wrangler/src/__tests__/helpers/end-event-loop.ts

```diff
@@ -0,0 +1,3 @@
+export function endEventLoop() {
```

Comment:
This is probably simple but I'm not super familiar with event loops so a docstring explaining what it does (and maybe why you'd use it) would be helpful

Reply:
✅